client.rb

```diff
@@ -0,0 +1,30 @@
+require 'knackhq/client/version'
```

Comment:
Thanks. This is a strange one. I thought I did a git remove on this file...Atom puts files you have open back in the directory even if you delete the file aparently.